
See:
	gtk-integration-salinst.diff for salinst (locking) changes.
	gtk-integration-saldisp.diff for event handling changes.


Index: vcl/util/makefile.mk
===================================================================
RCS file: /cvs/gsl/vcl/util/makefile.mk,v
retrieving revision 1.46.2.1
diff -u -p -u -r1.46.2.1 makefile.mk
--- vcl/util/makefile.mk	21 Jul 2003 14:48:45 -0000	1.46.2.1
+++ vcl/util/makefile.mk	13 Aug 2003 15:36:19 -0000
@@ -313,6 +317,10 @@ SHL1STDLIBS += -laudio
 SHL1STDLIBS += -ldl -lnsl -lsocket
 .ENDIF # SOLARIS
 .ENDIF          # "$(OS)"=="LINUX" || "$(OS)"=="SOLARIS" || "$(OS)"=="FREEBSD"
+
+.IF "$(GUIBASE)"=="unx"
+SHL1STDLIBS += `pkg-config --libs gtk+-2.0`
+.ENDIF
 
 .ENDIF          # "$(GUI)"=="UNX"
 
Index: vcl/unx/source/app/makefile.mk
===================================================================
RCS file: /cvs/gsl/vcl/unx/source/app/makefile.mk,v
retrieving revision 1.9.232.1
diff -u -p -u -r1.9.232.1 makefile.mk
--- vcl/unx/source/app/makefile.mk	21 Jul 2003 14:48:43 -0000	1.9.232.1
+++ vcl/unx/source/app/makefile.mk	14 Aug 2003 16:07:07 -0000
@@ -128,6 +131,8 @@ SLOFILES+=$(SLO)$/getfpsoli.obj
 SLOFILES=\
 			$(SLO)$/salmain.obj
 .ENDIF
+
+CFLAGS+=`pkg-config --cflags gtk+-2.0`
 
 .IF "$(remote)"!=""
 EXCEPTIONSFILES=$(SLO)$/salmain.obj	\
